demos/gaze_estimation_demo/cpp_gapi/src/custom_kerenels.cpp

```diff
@@ -0,0 +1,200 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include "kernel_packages.hpp"
+#include "custom_kernels.hpp"
+
+#include <opencv2/imgproc.hpp>
+
+using namespace gaze_estimation;
+
+struct Estimators {
+    std::shared_ptr<HeadPoseEstimator>  headPoseEstimator;
+    std::shared_ptr<LandmarksEstimator> landmarksEstimator;
+    std::shared_ptr<EyeStateEstimator>  eyeStateEstimator;
+    std::shared_ptr<GazeEstimator>      gazeEstimator;
+};
+
+Estimators estimators;
+
+void rotateImageAroundCenter(const cv::Mat& srcImage,
+                             const float angle,
+                                   cv::Mat& dstImage) {
+    const auto width = srcImage.cols;
+    const auto height = srcImage.rows;
+
+    const cv::Size size(width, height);
+
+    const cv::Point2f center(static_cast<float>(width / 2), static_cast<float>(height / 2));
+
+    const auto rotMatrix = cv::getRotationMatrix2D(center, static_cast<double>(angle), 1);
+    cv::warpAffine(srcImage, dstImage, rotMatrix, size, 1, cv::BORDER_REPLICATE);
+}
+
+inline void toCHW(const cv::Mat& src, cv::Mat& dst) {
```

Comment:
Just curious, why `inline`? Or why `inline` only here.

Reply:
Simple function. Removed as it raises questions.